source/_drafts/The-Importance-of-Metadata.md

```diff
@@ -0,0 +1,18 @@
+---
+title: The Importance of Metadata
+author: Mukund Srinivas
+date:
+categories: general
+number:
+tags:
+---
+Agricultural technology(Agri-Tech) is a portfolio of products and serivices that help the agricultural industry, these could range from products that help in reducing operational costs of the client(developing algorithms for efficiently moving and storing food), to make policy decisions (What are the crops that need a subsidy that particular season).
```

Comment:
Agricultural technology (Agri-Tech) is a portfolio of products and services that help the agricultural industry, ranging from helping in reducing costs (efficiently moving and storing food) to making policy decisions (providing subsidies for crops).